org.eclipse.m2e.pde.connector/src/org/eclipse/m2e/pde/connector/PDEMavenBundlePluginConfigurator.java

```diff
@@ -0,0 +1,118 @@
+package org.eclipse.m2e.pde.connector;
```

Comment:
Missing copyright header.

Reply:
The project contains a feature file (that is the one published and distributed to the catalouge) that mention the EPL 1.0

https://github.com/tesla/m2eclipse-tycho/blob/master/org.sonatype.tycho.m2e.feature/license.html

Reply:
If git is right, the only contributor is @ifedorenko maybe he can claim here if it is EPL licensed?

```
git shortlog -n -s -- PDEMavenBundlePluginConfigurator.java
     5  Igor Fedorenko
```

Reply:
my fault, i missed to copy the header while migration the file (no idea why it was lost...) I will force push it...